modules/slacker-api/src/main/java/com/labs2160/slacker/api/ConfigurableRaw.java

```diff
@@ -0,0 +1,7 @@
+package com.labs2160.slacker.api;
+
+import java.util.Properties;
+
+public interface ConfigurableRaw {
```

Comment:
Having two interfaces, Configurable and ConfigurableRaw and the only difference is that one of the functions has an extra param smells like bad code.

For now, why not just add a new interface called "ResourceAware" with a function setResources(...)

Any class that needs both can implement both Configurable and ResourceAware.